packages/main/src/features/InputSuggestions.js

```diff
@@ -502,7 +502,7 @@ class Suggestions {
 	}
 
 	sanitizeText(text) {
-		return text && text.replace("<", "&lt");
+		return encodeXML(text);
```

Comment:
Why are we encoding HTML content as XML?

Perhaps it will work, but I think it's a bit over kill for what you need to actually do.